core/model.py

```diff
@@ -56,23 +44,32 @@ def __init__(self, model_file=PATH_TO_CKPT, label_file=PATH_TO_LABELS):
         self.category_index = category_index
         self.categories = categories
 
-    def predict(self, imageRaw, threshold):  #was origninally run_inference_for_single_image
-        image = preprocess_image(imageRaw)
+    def _read_image(self, image_data):
+        image = Image.open(io.BytesIO(image_data)).convert("RGB")
+        return image
+
+    def _pre_process(self, image):
+        (im_width, im_height) = image.size
+        return np.array(image.getdata()).reshape(
+            (im_height, im_width, 3)).astype(np.uint8)
+
+    def _post_process(preds):
+        pass
```

Comment:
This should return `preds` if it's not being used.

Reply:
I just delete the function